src/components/forms/__tests__/PhoneInput.test.tsx

```diff
@@ -0,0 +1,137 @@
+import * as React from "react"
+import { screen, render } from "@testing-library/react"
+import user from "@testing-library/user-event"
+
+import { PhoneInput, CountryCode } from "../PhoneInput"
+
+describe("PhoneInput", () => {
+  const onChange = jest.fn()
+
+  afterEach(() => onChange.mockClear())
```

Comment:
can we move this to `jest.setup.ts`, namely `jest.clearAllMocks()`. sometimes this can cause issues in established test suites, but given this library is young and we want to write true unit tests, ensuring mocks are reset between each unit is a good idea